integ-test/src/test/java/org/opensearch/sql/legacy/CursorIT.java

```diff
@@ -182,7 +182,7 @@ public void validTotalResultWithAndWithoutPaginationOrderBy() throws IOException
     String selectQuery =
         StringUtils.format(
             "SELECT firstname, state FROM %s ORDER BY balance DESC ", TEST_INDEX_ACCOUNT);
-    verifyWithAndWithoutPaginationResponse(selectQuery + " LIMIT 2000", selectQuery, 26, false);
```

Comment:
I forgot whether this class is the main IT for pagination? Is the new PIT logic enabled to test now?

Reply:
This class is only responsible for legacy/v1 engine pagination queries. For v2 engine queries, there is a different base class. Currently, integ-test cluster does not set the feature flag settings and by default the cluster settings will have PIT feature flag enabled/set to true. Hence there are around 80+ tests testing new PIT changes.

Reply:
Not urgent but I think it would be helpful to get the IT run with PIT and Scroll both. In case we need to rollback to Scroll if anything wrong.

Reply:
Actually we are not disabling/removing Scroll implementation. There is a cluster level settings called `SQL_PAGINATION_API_SEARCH_AFTER` which is set to true to use PIT or false to default to Scroll API. So current IT tests run based on the default setting of the above key and it covers both Scroll and PIT.

Reply:
Do you mean the same IT set run 1 pass for PIT and another pass for Scroll?

Reply:
oh you meant running 2 passes of same set of tests for both PIT and Scroll? Ok I think that can be done by modifying the integ-test cluster settings as required. But currently we do not explicitly set the API settings in the test-cluster, so by default PIT changes are covered as part of IT tests and only 1 pass is run. I'll add this as part of the changes here - https://github.com/opensearch-project/sql/issues/2945